.github/workflows/lock.yml

```diff
@@ -0,0 +1,16 @@
+name: 'Lock Threads'
+
+on:
+  schedule:
+    - cron: '0 * * * *'
```

Comment:
The action has a limit of 50 issues per run, so lets run it once every hour for now. We'll change it to once a day later.